apisix/init.lua

```diff
@@ -107,6 +107,10 @@ function _M.http_init_worker()
     require("apisix.debug").init_worker()
     require("apisix.upstream").init_worker()
 
+    -- need call seed
+    local uuid = require("resty.jit-uuid")
+    uuid.seed()
```

Comment:
https://github.com/apache/apisix/blob/master/apisix/init.lua#L72

We have set the seed with a better way. So we do not need to set it agian.

Reply:
But in k8s, it not work very well.

Reply:
you can print more log for confirming why it works badly. we need to know the reason.

Reply:
@ zlm0125  ping